docs/operator-docs/granule-workflows.md

```diff
@@ -0,0 +1,95 @@
+---
+id: granule-workflows
+title: Granule Workflows
+hide_title: true
+---
+
+# Granule Workflows
+
+## Ingest Kinesis
+
+1. Copy your AWS Long Term Access Key.
+
+2. Add a Collection
+
+* Open another tab in your browser and go to your Earthdata Git repository URL or where your collection data files are located. If you do not have collection data stored, you can manually configure your collections as well.
+> **Tip**: A list of collection config params can be accessed [here] (https://nasa.github.io/cumulus/docs/data-cookbooks/setup#collections)
```

Comment:
The link is not showing up correctly 
![image](https://user-images.githubusercontent.com/25331898/87804777-59d81c80-c822-11ea-9247-463eb43f4713.png)

Reply:
Hmm, I'm not seeing that. 
<img width="1113" alt="Screen Shot 2020-07-17 at 12 12 47 PM" src="https://user-images.githubusercontent.com/40603352/87807941-e71d7000-c826-11ea-8d7c-d21c8397dd25.png">